src/main/java/com/epam/ivankaliada/BasicSpringApp/entity/Employee.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.ivankaliada.BasicSpringApp.entity;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.util.Date;
+
+@Data
```

Comment:
Be careful with using `@Data` annotation with JPA entities.
https://jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/